City-of-Pori/pori-kada
# PORI-86 && 98: Theme page visitpori and basic page.

Theme page visitpori structure and styling. Some minor basic page styling.

To test:
You need few filler pages with visitpori domain access set and some visitpori theme. IF you dont have any visitpori theme items, go add them in taxonomy under visitpori_theme
1. drush updb -y && drush fra -y && drush cc all
2. go to your local site and make sure you have visitpori theme on your local.visitpori.fi
3. go and create a page, add: 
- a title 
- liftup image
- body text
- set a visitpori theme (if you dont have any, you need to go and add them in taxonomy under visitpori_theme, set a color for the theme).
- Add few topics in the topics tab
- in widgets check the theme mainpage checkbox.
- Add few subpages (create few filler pages with visitpori domain access and some theme)
- set domain access to visitpori
4. Confirm that the theme page looks good. (NOTICE the image is not styled to go under the text content since it proved to be a bit of a challenge and to save time this is not done now.)
5. Go edit the filler pages, add some body text and set a visitpori theme, Add some topics and additional-info links
6. Go to that page and make sure it looks good.
